models/issue_mail.go

```diff
@@ -16,7 +16,7 @@ import (
 )
 
 func (issue *Issue) mailSubject() string {
-	return fmt.Sprintf("[%s] %s (#%d)", issue.Repo.Name, issue.Title, issue.Index)
+	return fmt.Sprintf("[%s/%s] %s (#%d)", issue.Repo.OwnerName, issue.Repo.Name, issue.Title, issue.Index)
```

Comment:
Are we sure that Repo.OwnerName is definitely set? If not we might need:

```suggestion
	return fmt.Sprintf("[%s/%s] %s (#%d)", issue.Repo.MustOwnerName(), issue.Repo.Name, issue.Title, issue.Index)
```

But then you need to check that we're not in a session otherwise we might deadlock.